test/functional/mempool_datacarrier.py

```diff
@@ -51,6 +66,20 @@ def run_test(self):
         too_long_data = random_bytes(MAX_OP_RETURN_RELAY - 2)
         small_data = random_bytes(MAX_OP_RETURN_RELAY - 4)
 
+        small_annex = random_bytes(0)
+        large_annex = random_bytes(MAX_ANNEX_DATA)
+        oversize_annex = random_bytes(127)
```

Comment:
Maybe `MAX_ANNEX_DATA + 1` here, rather than a fixed number, as if the max was raised this would no longer be oversized?